src/routes/productRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const productController = require("../controllers/productController");
+const token = require("../middlewares/verfiyToken")
+
+router.get("/detail", token.verfiyToken, productController.detailPage)
+router.post("/addProducts", token.verfiyToken, productController.addProducts);
```

Comment:
```suggestion
router.post("/", token.verfiyToken, productController.addProducts);
```
post HTTP 메소드만으로도 제품을 생성한다는 뜻이 통하기 때문에 uri에 동사를 제거하셔도 됩니다